presto-main/src/main/java/com/facebook/presto/operator/SharedLookupSource.java

```diff
@@ -30,11 +30,18 @@
     @GuardedBy("this")
     private boolean freed;
 
-    public SharedLookupSource(LookupSource lookupSource, TaskContext taskContext)
+    public SharedLookupSource(LookupSource lookupSource, OperatorContext operatorContext)
```

Comment:
I think there is a new one of these for cross joins that will need to be updated also.